src/inject/dynamic-theme/style-manager.ts

```diff
@@ -23,8 +23,9 @@ declare global {
 
 export type StyleElement = HTMLLinkElement | HTMLStyleElement;
 
+export type detailsArgument = {secondRound: boolean};
```

Comment:
Wonderful! One note: Uppercase names for types.

Reply:
Will be remembered for the next time!